resources/js/app.js

```diff
@@ -1001,6 +1001,20 @@ function datePickerChart(){
 	   }
 	 });
 }
+
+var date = new Date();
+var tdate = date.getDate();
+if (tdate < 10) {
+	tdate = "0" + tdate;
+}
+var month = date.getMonth() + 1;
+if (month < 10) {
+	month = "0" + month;
+}
+var year = date.getUTCFullYear();
+var minDate = year + "-" + month + "-" + tdate;
+document.getElementById("start_date").setAttribute("min", minDate);
+document.getElementById("end_date").setAttribute("min", minDate);
```

Comment:
Can you add a check here for the page where you need to load this script. You just need to add an id on the page and check the length of the id like this:

```suggestion
if ($('#page_id').length) {
    var date = new Date();
    var tdate = date.getDate();
    if (tdate < 10) {
                tdate = "0" + date;
    }
    var month = date.getMonth() + 1;
    if (month < 10) {
        	month = "0" + month;
    }
    var year = date.getUTCFullYear();
    var minDate = year + "-" + month + "-" + date;
    document.getElementById("start_date").setAttribute("min", minDate);
    document.getElementById("end_date").setAttribute("min", minDate);
}
```

Reply:
@GauravGusain98 I have made the required changes. And for the id part, I have used a unique id for start and end dates.These are not generic for the entire hr module. The previous dates are not visible for job description pages only.